### PR TITLE
WIP: Move to runtime 18.08

### PR DESCRIPTION
**Pull request by ananace**
Just leaving this here to stop forgetting about it.

### CODE REVIEW

**Comment by TingPing** [comment]
bot, build

**Comment by ananace** [comment]
Already know that it's broken, something with qtt

**Comment by TingPing** [comment]
I'd try waiting for the next KDE runtime based on 18.08 then.

**Comment by ananace** [comment]
When that appears certainly, but until then I should still upgrade the other deps - and maybe see if a newer Qt4 works with 18.08 (OpenMW technically doesn't support Qt5 yet, even though it *works* with it)

File: org.openmw.OpenMW.yaml [hunk 1]
```diff
@@ -116,12 +116,24 @@ modules:
       - examples
     build-options:
       env:
-        CXXFLAGS: "-std=gnu++98"
+        CXXFLAGS: "-std=gnu++98 -O2"
```
Comment (TingPing) [inline org.openmw.OpenMW.yaml]:
```yaml
  build-options:
    cxxflags: -std=gnu++98
```

(Note you don't need to add -O2, its in default flags)

Comment (ananace) [reply org.openmw.OpenMW.yaml]: Just copied the exact changes from the shared module, but I agree that the `-O2` felt a bit out of place

Comment (TingPing) [reply org.openmw.OpenMW.yaml]: Ah, the shared-module has a lot of extra complexity becuese it has to support 1.6 and 18.08, like the conditional patch command.

Comment (TingPing) [reply org.openmw.OpenMW.yaml]: Why exactly aren't you using the shared module?

**Pull request closed without merging**